scripts/setup_host.sh

```diff
@@ -141,7 +145,16 @@ function ubu_install_qemu_gvt(){
     sudo apt install -y git libfdt-dev libpixman-1-dev libssl-dev vim socat libsdl2-dev libspice-server-dev autoconf libtool xtightvncviewer tightvncserver x11vnc uuid-runtime uuid uml-utilities bridge-utils python-dev liblzma-dev libc6-dev libegl1-mesa-dev libepoxy-dev libdrm-dev libgbm-dev libaio-dev libusb-1.0-0-dev libgtk-3-dev bison libcap-dev libattr1-dev flex libvirglrenderer-dev build-essential gettext libegl-mesa0 libegl-dev libglvnd-dev libgl1-mesa-dev libgl1-mesa-dev libgles2-mesa-dev libegl1 gcc g++ pkg-config libpulse-dev libgl1-mesa-dri libdaxctl-dev libseccomp-dev
     sudo apt install -y ninja-build libcap-ng-dev
 
-    [ ! -f $CIV_WORK_DIR/$QEMU_REL.tar.xz ] && wget https://download.qemu.org/$QEMU_REL.tar.xz -P $CIV_WORK_DIR
+    #Create QEMU_CACHE_DIR if it doesnt exists
+    mkdir -p $QEMU_CACHE_DIR
+
+    #Download QEMU_REL.tar.xz if it doesnt exist in QEMU_CACHE_DIR
+    
+    [ ! -f $QEMU_CACHE_DIR/$QEMU_REL.tar.xz ] && check_qemu_network && wget https://download.qemu.org/$QEMU_REL.tar.xz -P $QEMU_CACHE_DIR
+
+    # Copy QEMU_REL.tar.xz from QEMU_CACHE_DIR to CIV_WORK_DIR
+    cp $QEMU_CACHE_DIR/$QEMU_REL.tar.xz $CIV_WORK_DIR
```

Comment:
why need a copy? suggest to use `tar -xf $QEMU_CACHE_DIR/$QEMU_REL.tar.xz -C $CIV_WORK_DIR`

Reply:
Right, let me fix this. Thanks for the suggestion.